src/ngraph/runtime/gpu/gpu_emitter.cpp

```diff
@@ -571,9 +571,64 @@ cudnnSetOpTensorDescriptor(opTensorDesc,
                 // Other cases (reordering of axes for tensors with rank>2) are not handled yet.
```

Comment:
Change this comment. 

Also, I noticed an extra indent that should be removed on line (556) - https://github.com/NervanaSystems/ngraph/blob/61623fb00f93b4d95d259337cf03ec6b2d3a286b/src/ngraph/runtime/gpu/gpu_emitter.cpp#L556

Reply:
Thanks, updated.